file/validate.go

```diff
@@ -1,13 +1,71 @@
 package file
 
 import (
+	"encoding/json"
 	"fmt"
+	"strings"
 
 	ghodss "github.com/ghodss/yaml"
 	"github.com/kong/deck/utils"
 	"github.com/xeipuuv/gojsonschema"
 )
 
+func check(item string, t *string) string {
+	if t != nil || len(*t) > 0 {
+		fmt.Println(item + "[err] is not allowed to be specified.")
```

Comment:
With the problem items now collected in the error, you'll want to remove this to avoid printing duplicate information:
```suggestion
```